src/lib/network/network_autoconfiguration.rb

```diff
@@ -31,16 +31,17 @@ class NetworkAutoconfiguration
     include Singleton
     include Logger
 
-    Yast.import "Lan"
-    Yast.import "NetworkInterfaces"
-    Yast.import "NetworkService"
-    Yast.import "Package"
-    Yast.import "DNS"
-    Yast.import "Arch"
-    Yast.import "Host"
-
     BASH_PATH = Path.new(".target.bash")
 
+    def initialize
+      Yast.import "Lan"
+      Yast.import "NetworkService"
```

Comment:
this one seems to be useless too as the respective code uses new "ProposalSettings" class